src/gsi-ncdiag/gsi_ncdiag.py

```diff
@@ -839,7 +864,7 @@ def toIODAobs(self, OutDir, clobber=True, platforms=None):
                         obstimes = [self.validtime + dt.timedelta(hours=float(tmp[a])) for a in range(len(tmp))]
                         obstimes = [a.strftime("%Y-%m-%dT%H:%M:%SZ") for a in obstimes]
                         outdata[(loc_mdata_name, 'MetaData')] = np.array(obstimes, dtype=object)
-                        varAttrs[(loc_mdata_name, 'MetaData')]['units'] = 'UTC Time in YYYY-MM-DDTHH:MM:SSZ format'
+                        varAttrs[(loc_mdata_name, 'MetaData')]['units'] = 'seconds since 1970-01-01T00:00:00Z'
```

Comment:
Suggest removing this, some folks have had troubles specifying this when running the tests.